README.md

```diff
@@ -1 +1,140 @@
+
+### Average Time On Page
+Average amount of time users spent viewing a page as defined in 
+[Google Analytics Core API](https://developers.google.com/analytics/devguides/reporting/core/dimsmets#view=detail&group=page_tracking&jump=ga_avgtimeonpage)
+
+Preferred over [time on page](https://developers.google.com/analytics/devguides/reporting/core/dimsmets#view=detail&group=page_tracking&jump=ga_timeonpage) 
+which does not apply to exit pages.
+
+## Unique Pageviews
+The number of different (unique) pages within a session as defined in 
+[Google Analytics Core Reporting API](https://developers.google.com/analytics/devguides/reporting/core/dimsmets#view=detail&group=page_tracking&jump=ga_uniquepageviews)
+
+A session according to [how visits are calculated in Google Analytics](https://support.google.com/analytics/answer/2731565)
+expires after 30 minutes of inactivity or at midnight.
+
+# Interpretation
+## Top Left Quarter: Challenge
+_Enough people_ visit that page but _read less than 50%_ of its content:
+
+- Poor readability score? Would simplifying the language make it better?
+- Length of content? Would splitting the content into multiple pages make people read it?
+- Too many internal links? Do people jump to a different page without reading?
+- Too many external links? Do people jump to an external site?
+- Too many examples? Too many examples on that page which increase the expected reading time?
+- Does the page also have high exit rates?
+
+## Top Right Quarter: Great
+_Enough people_ visit that page and do _read more than 50%_ of its content:
+
+- This is where all of the core content should end up
+
+## Bottom Right Quarter: Good
+People who visit that page _read more than 50%_ of its content but _not enough people_ visit it:
+
+- It it a page of special interest that justifies this, e.g. about divorce?
+- Should this page be made more prominent by linking to it more often?
+- Should online marketing be targeting this page to increase its traffic?
+
+## Bottom Left Quarter: Bad
+_Not enough people_ visit that page and those who do _read less than 50%_ of its content:
+
+- The worst place to be..
+- Improve the percentage of content read on that page before trying to increase its traffic
+
+## Limitations
+
+The homepage is excluded from the Content Performance chart as it warrants special analysis.
+
+Expected Reading Time does not take into account:
+
+- Page elements such as headers, footers, images, and side content that incur cognitive load on a user and hence require extra time to process
+- Reading score as a factor of content complexity that could impact reading time
+- Repeat visits to the same page during a single session
+
+# Usage
+Collect data:
+
+```ruby
+rake collect
+```
+
+Start the app:
+
+```sh
+$ ./bin/foreman s
+```
+
+# Google Developers Console
+[Pension Wise Google Developers Console]
+
+# Dependencies
+* [Sinatra]
```

Comment:
You don't need to list this here - that's what the Gemfile gives us. In order to use the Gemfile however we need Ruby and Bundler, so we should detail those as dependencies.